tests/ext2/plain/ext2.c

```diff
@@ -20,6 +20,20 @@ gid_t myst_syscall_getegid(void)
     return getegid();
 }
 
+int check_thread_group_membership(gid_t group)
```

Comment:
Is this needed?

Reply:
These are stubs so that this test can build. Both `check_thread_group_membership` and `myst_thread_self` are defined in libmystkernel.so, which this test doesn't link against.

Reply:
I am confused. Why would a test program introduce dependency on mystikos internal functions?

Reply:
This PR added these two functions to be called from the ext2 implementation. The test doesn't exercise these functions, but it relies on libmystext2.a, and that in turn relies of definition of these functions.